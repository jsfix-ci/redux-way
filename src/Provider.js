import React from 'react'
import PropTypes from 'prop-types'

export default class Provider extends React.Component {
	static propTypes = {
		store: PropTypes.object.isRequired
	}

	componentWillMount() {
		
	}
}