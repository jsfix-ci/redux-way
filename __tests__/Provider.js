import React from 'react'
import ReactTestRenderer from 'react-test-renderer';
import Provider from '../src/Provider';

describe('Provider', () => {
	it('must ', () => {
		const store = {};
		const renderer = ReactTestRenderer.create(
			<Provider store={store}>
				<div></div>
			</Provider>
		).toJSON();

		expect(renderer).toMatchSnapshot();
		
	});
})