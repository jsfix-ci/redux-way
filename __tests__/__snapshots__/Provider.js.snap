// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider must  1`] = `<div />`;
